packages/beacon-node/src/network/discv5/index.ts

```diff
@@ -42,7 +42,7 @@ export class Discv5Worker extends (EventEmitter as {new (): StrictEventEmitter<E
   static async init(opts: Discv5Opts): Promise<Discv5Worker> {
     const workerData: Discv5WorkerData = {
       enr: opts.discv5.enr,
-      peerIdProto: exportToProtobuf(opts.peerId),
+      peerIdProto: exportToProtobuf(opts.peerId as Secp256k1PeerId),
```

Comment:
Any way we could avoid this cast?

Reply:
unfortunately not, see https://github.com/libp2p/js-libp2p/pull/2496